src/controller_examples/rabbitmq_controller/proof/liveness.rs

```diff
@@ -171,8 +172,8 @@ proof fn invariants_is_stable(rabbitmq: RabbitmqClusterView)
 // Note that different from the above invariants, these do not hold for the entire execution from init.
 // They only hold since some point (e.g., when the rest id counter is the same as rest_id).
 // Some of these invariants are also based on the assumptions.
-spec fn invariants_since_rest_id(rabbitmq: RabbitmqClusterView, rest_id: RestId) -> TempPred<ClusterState> {
-    always(lift_state(RMQCluster::rest_id_counter_is_no_smaller_than(rest_id)))
+spec fn invariants_since_rest_id(rabbitmq: RabbitmqClusterView, rest_id: RestId) -> TempPred<RMQCluster> {
+    always(lift_state(RMQCluster::rest_id_counter_is_no_smaller_than_state_pred(rest_id)))
```

Comment:
What is this renaming for (_state_pred)?

Reply:
There’s a method with the same name, originally implemented for State

Reply:
Why not we kill the definition of `rest_id_counter_is_no_smaller_than` and for each usage we either inline it or replace it with `rest_id_counter_is_no_smaller_than_state_pred`?

Reply:
And then remove `_state_pred` from its name

Reply:
That’s just an alternative. But we have to think about how to name this because it’s common to have two methods with the similar functions. That’s also why I keep both of the two without making one of them inline.

Reply:
Since we use the StatePred version a lot and they usually appear at the surface of each lemma (pre and post conditions), we should always assign the better name (the shorter one) to the StatePred version.
For their bool siblings, maybe call it `XXX_bool`?